composer.json

```diff
@@ -22,6 +22,7 @@
     "easyframework/collections": "dev-master"
   },
   "require-dev": {
+    "php-amqplib/php-amqplib": "2.6.*",
```

Comment:
This is actually debatable. Whoever uses this library can opt for not want to install amqp library. Once we put this here it installs it by default (As composer installs the dev requirements by default). I think who implement this library needs to add one of the suggestions offered on the suggestion section.